src/legacy/plugin_discovery/types.ts

```diff
@@ -66,6 +66,8 @@ export interface LegacyPluginOptions {
     home: string[];
     mappings: any;
     savedObjectSchemas: SavedObjectsSchemaDefinition;
+    savedObjectsManagement: any;
```

Comment:
Is there a better way to introduce this definition than `any`? Maybe use `unknown` so people are aware it has no capabilities?

Reply:
Good catch! I found an actual typing definition for this.